website/docs/cli/commands/completion/atmos-completion-bash.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: atmos completion bash
+sidebar_label: bash
+sidebar_class_name: command
+description: Use this command to generate completion scripts for `Bash`
+---
+
+
+## Bash Completion Setup
```

Comment:
I think we can just use tabs, instead of adding more pages.

https://docusaurus.io/docs/3.1.1/markdown-features/tabs

Plenty of examples like https://atmos.tools/install

Reply:
Done